common_utils/context.go

```diff
@@ -55,8 +65,25 @@ func GetContext(ctx context.Context) (*RequestContext, context.Context) {
 
 func GetUsername(ctx context.Context, username *string) {
 	rc, _ := GetContext(ctx)
-        if rc != nil {
-            *username = rc.Auth.User
-        }
+	if rc != nil {
+		*username = rc.Auth.User
+	}
+}
+
+func InitCounters() {
+	for i := 0; i < len(CountersName); i++ {
+		globalCounters[i] = 0
+	}
+	SetMemCounters(&globalCounters)
+}
+
+func IncCounter(name string) {
```

Comment:
Why use string as an index? If an enum were used then there would be no need for the loop in line 81, right?